src/components/FeatureBannersData.js

```diff
@@ -0,0 +1,11 @@
+import featuredBannersData from '../mocks/en-us/featured-banners.json'
```

Comment:
For Featured Banners, products and categories maybe they are not really components and more of a utility function